test/test-visualizer.js

```diff
@@ -101,7 +101,8 @@ test('simple parse tree', function(t) {
   ohmEditor.registerEvents({
     'parse:input': ['matchResult', 'trace'],
     'change:inputEditor': ['codeMirror'],
-    'change:grammarEditor': ['codeMirror']
+    'change:grammarEditor': ['codeMirror'],
+    'change:action': ['trace']
```

Comment:
Is this something that effects other parts of the editor? If so, then it's fine to put it here. But if it's specific to the semantics editor, then it probably belongs somewhere else. E.g., maybe it makes sense on `ohmEditor.parseTree`...or maybe we should create create a new CheckedEmitter named `ohmEditor.semantics`.

And it would be good to have a bit more descriptive of a name...with no other context, "action" is pretty ambiguous. What about "change:semanticOperation" (isn't it the _operation_ that's changing)?